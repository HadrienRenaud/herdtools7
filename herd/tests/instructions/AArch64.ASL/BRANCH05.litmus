AArch64 BRANCH05
{
0:X0=1;
0:X1=1;
}

 P0           ;
CMP W1,#1     ;
B.NE .+8      ;
ADD W0,W0,#1  ;
forall 0:X0=2
