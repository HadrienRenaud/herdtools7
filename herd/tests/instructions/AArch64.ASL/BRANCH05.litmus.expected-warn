File ./herd/libdir/asl-pseudocode/shared_pseudocode.asl, line 10960,
  characters 19 to 26:
Warning: Removing some values that would fail with op << from constraint set
{-1..6} gave {0..6}. Continuing with this constraint set.
File ./herd/libdir/asl-pseudocode/shared_pseudocode.asl, line 10960,
  characters 19 to 26:
Warning: Removing some values that would fail with op << from constraint set
{-1..6} gave {0..6}. Continuing with this constraint set.
File ./herd/libdir/asl-pseudocode/shared_pseudocode.asl, line 10960,
  characters 19 to 26:
Warning: Removing some values that would fail with op << from constraint set
{-1..6} gave {0..6}. Continuing with this constraint set.
