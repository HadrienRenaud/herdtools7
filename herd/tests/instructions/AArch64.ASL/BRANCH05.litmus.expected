Test BRANCH05 Required
States 1
0:X0=2;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X0=2)
Observation BRANCH05 Always 1 0
Hash=37cc9ba415441cd0389f9feec2014fe9

